keyform/static/keyform/js/add_form.js

```diff
@@ -13,13 +13,13 @@ $(document).ready(function() {
         }
     });
 
-    function mailbox_field() {
+    function hide_keytype_field() {
```

Comment:
Technically we're hiding the core number field, not the key type...

Reply:
It's a field contained in the KEY TYPE box, so this code can be used in a more vague sense in the future

Reply:
Ah, I see. Well, that would be the KeyData box in that case, not KeyType. But I like where your head's at.

Reply:
Would hide_keydata_field() be acceptable?

Reply:
It would.